### ISSUE TITLE
Zapytania o niestandardowe atrybuty

### ISSUE THREAD

**Issue by fjn-6666** [allegro/allegro-api#3357]
Witam
    Część katalogu platformy obsługuje dostosowywanie 4 atrybutów,
     Czy mogę używać interfejsu API do dostosowywania nazw i wartości atrybutów?
![atrybut](https://user-images.githubusercontent.com/66991708/85488876-a6548180-b601-11ea-8ce5-395622c638d8.png)

**Comment by MaciejFrackowiak** [comment]
Nie, parametry własne dostępne są obecnie tylko na stronie WWW, nie ma ich w API.

**Comment by Lebioda** [comment]
A czy i kiedy będą w API?

**Comment by MaciejFrackowiak** [comment]
@Lebioda Tak, mamy to na liście sugestii związanych z rozwojem API Allegro - w momencie wdrożenia rozwiązania zamieścimy osobny post na forum.
